src/components/ConfigureAutomatonWindow.tsx

```diff
@@ -73,12 +72,28 @@ function ListItem_TokenEditor(
  * @returns
  */
 function AlphabetList() {
-  const [alphabet, setAlphabet] = useState(StateManager.alphabet);
+  const [alphabet, setAlphabet] = useState<TokenWrapper[]>([]);
+  const [inputValue, setInputValue] = useState("");
 
   function addTokenToAlphabet() {
-    StateManager.addToken();
+    const newSymbols = new Set(
+      inputValue.split(",").map((token) => token.trim()),
```

Comment:
This allows tokens to be inputted whose labels are multiple characters long. Once they are created, their labels cannot be changed; they have to be deleted and then recreated. I'm guessing there is code that prevents modifications from going through if the resultant string would be longer than 1 character?